CONTRIBUTING.md

```diff
@@ -21,6 +22,44 @@ git remote add upstream https://github.com/FerretDB/FerretDB.git
 
 ### Setting up the development environment
 
+In the github-actions there are checks that new code complies the imports sorted convention
+and that the code is gofmt-ed. To avoid failing PR checks, please setup your beloved text editor
```

Comment:
https://sembr.org

Reply:
fixed, added to the guideline.

Reply:
I’m pretty sure we still break rule 4 of it. Please use one sentence per line in all markdown files except the changelog. See also #396

Reply:
than you. Fixed sembr also for the `processes.md` file. Added pre-hook commit example.